### DISCUSSION TITLE
CSLA v5.5.2

### DISCUSSION THREAD

**Discussion by j055** [MarimerLLC/csla#2532] [Questions]
Hi Rocky

Any chance of getting v5.5.2 released to Nuget? It's got some fixes I could really do with at the moment.

Thank you
Andrew

**Comment by rockfordlhotka** [comment]
I could push a preview. I haven't had time to try and research why the resources aren't working in the last package.

**Reply by rockfordlhotka** [reply]
Done

**Reply by j055** [reply]
OK thanks, you mean this one: https://github.com/MarimerLLC/csla/issues/2498?

**Reply by rockfordlhotka** [reply]
Yes, usually it is a nuget packaging issue, but I see nothing obvious, so it'll take some serious time (I'm afraid) to figure out what's going on).
